crates/proto/src/gamepacket.rs

```diff
@@ -336,6 +338,7 @@ impl GamePacket {
     const SubChunkPacketID: u16 = 174;
     const SubChunkRequestPacketID: u16 = 175;
     const DimensionDataID: u16 = 180;
+    const ccID: u16 = 186;
```

Comment:
what is `cc`??

Reply:
"In the latest commit, this strange issue with 'cc' has been fixed."